### PR TITLE
Add resposive design to the index js file to work on mobile

### PR DESCRIPTION
**Pull request by mohamedlafiteh**

### CODE REVIEW

File: client/src/components/navbar/NavBarMenuItems.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            <Menu.Item
+              name="user registration"
+              as={Link}
+              to="/sign-up-user"
+              header
+              active={activeItem === "user registration"}
+              onClick={this.handleItemClick}
+            />
+            <Menu.Item
+              name="interpreter registration"
```
Comment (llh1) [inline client/src/components/navbar/NavBarMenuItems.js]: I think this was renamed to "translator registration" in master branch, can you make it the same?

File: client/src/components/navbar/NavBarMenuItems.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+              as={Link}
+              to="/sign-up-interpreter"
+              header
+              active={activeItem === "interpreter registration"}
```
Comment (llh1) [inline client/src/components/navbar/NavBarMenuItems.js]: "translator registration"

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/navbar/NavBarMenuItems.js (hunk 1 above)
Comment (mohamedlafiteh) [reply client/src/components/navbar/NavBarMenuItems.js]: I fixed them

File: client/src/components/navbar/NavBarMenuItems.js (hunk 2 above)
Comment (mohamedlafiteh) [reply client/src/components/navbar/NavBarMenuItems.js]: fixed now

**Pull request merged**